cypress-tests/cypress/e2e/PaymentUtils/Noon.js

```diff
@@ -0,0 +1,693 @@
+        },
+      },
+    },
+    Refund: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "AED",
+        customer_acceptance: null,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "succeeded",
+        },
+      },
+    },
+    manualPaymentRefund: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "AED",
+        customer_acceptance: null,
+      },
+      Response: {
+        status: 200,
+        trigger_skip: true,
+        body: {
+          status: "failed",
+        },
+      },
+    },
+    manualPaymentPartialRefund: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        customer_acceptance: null,
+      },
+      Response: {
+        status: 200,
+        trigger_skip: true,
+        body: {
+          status: "failed",
+        },
+      },
+    },
+    PartialRefund: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        customer_acceptance: null,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "succeeded",
+        },
+      },
+    },
+    SyncRefund: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        customer_acceptance: null,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "succeeded",
+        },
+      },
+    },
+    MandateSingleUse3DSAutoCapture: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulThreeDSTestCardDetails,
+        },
+        connector_metadata: connectorMetadata,
+        mandate_data: singleUseMandateData,
+      },
+      Response: {
+        status: 200,
+        trigger_skip: true,
+        body: {
+          status: "requires_customer_action",
+        },
+      },
+    },
+    MandateSingleUse3DSManualCapture: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulThreeDSTestCardDetails,
+        },
+        mandate_data: singleUseMandateData,
+      },
+      Response: {
+        status: 200,
+        body: {
+          status: "requires_customer_action",
+        },
+      },
+    },
+    MandateSingleUseNo3DSAutoCapture: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "AED",
+        mandate_data: singleUseMandateData,
+        connector_metadata: connectorMetadata,
+      },
+      Response: {
+        status: 200,
+        trigger_skip: true,
```

Comment:
I would suggest you moving this to new object called as `Configs`.

Please check my draft PR for reference.

Reply:
We can make this change when your pr gets merged